e2e/internal/client/cli.go

```diff
@@ -295,7 +324,15 @@ func (cli *CLI) EnvInit(opts *EnvInitRequest) (string, error) {
 		commands = append(commands, "--prod")
 	}
 	if !opts.CustomizedEnv {
-		commands = append(commands, "--no-custom-resources")
```

Comment:
Can we not merge this change separately which would be super small than the rest of the tests? This way we can have passing e2e tests